src/ol/structs/lrucache.js

```diff
@@ -230,6 +230,18 @@ ol.structs.LRUCache.prototype.pop = function() {
  * @param {string} key Key.
  * @param {T} value Value.
  */
+ol.structs.LRUCache.prototype.replace = function(key, value) {
+  var entry = this.entries_[key];
+  goog.asserts.assert(entry !== undefined, 'an entry exists for key %s', key);
+  this.get(key);
```

Comment:
`get` updates the LRU structure (`oldest`, `newest`, etc.).

Maybe this would be better:

```js
ol.structs.LRUCache.prototype.replace = function(key, value) {
  this.get(key);  // update `newest_`
  this.entries_[key] = value;
};
```